### ISSUE TITLE
Error message when 1/duration & d.rate too close

### ISSUE THREAD

**Issue by ejanderson1** [statnet/EpiModel#276]
When the rate of dissolution is too close to d.rate the error message reads:

```ln log(pg/(ps2-pg)): NaNs produced```

Needs to be updated to something more informative.

**Comment by smjenness** [comment]
This combo still results in an error:

```r
dissolution_coefs(~offset(edges), 60, 0.01)
```

**Comment by smjenness** [comment]
@andsv2 is currently working on this.

**Comment by andsv2** [comment]
> This combo still results in an error:
> 
> ```r
> dissolution_coefs(~offset(edges), 60, 0.01)
> ```

Apologies if this is a simple question, but is the above meant to work? The operand (60, 0.01) resolves to less than 0 and shouldn't work.

Also is there a point at which the adjusted coef. of dissolution is unreasonable? For example if coef.diss = 3 and adj. coef.diss = K >>3.

**Comment by smjenness** [comment]
No, the above should not work. Anything that currently produces the warning message:

```r
Warning message:
In log(pg/(ps2 - pg)) : NaNs produced
```

because the adjusted coefficient is `NaN` should error instead.

The adjusted coefficient may be much greater than the crude coefficient. For example, this is fine:

```r
> dissolution_coefs(~offset(edges), 50, 0.01)
Dissolution Coefficients
=======================
Dissolution Model: ~offset(edges)
Target Statistics: 50
Crude Coefficient: 3.89182
Mortality/Exit Rate: 0.01
Adjusted Coefficient: 9.190138
```

**Comment by smjenness** [comment]
Closed with #287
